releases/cockroachcloud-01102022.md

```diff
@@ -0,0 +1,21 @@
+---
+title: What's New in CockroachDB Cloud
+toc: true
+summary: Additions and changes in CockroachDB Cloud since December 6, 2021
+---
+
+## January 10, 2022
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+### General changes
+
+- New {{ site.data.products.dedicated }} clusters will now run [v21.2.3](v21.2.3.html).
+- {{ site.data.products.serverless }} clusters will now run CockroachDB [v21.2.0-beta.4](v21.2.0-beta.4.html).
+- The CockroachDB Docs navigation is now organized by user tasks instead of by product.
```

Comment:
```suggestion
- The CockroachDB documentation navigation is now organized by user task instead of by product for CockroachDB Serverless, CockroachDB Dedicated, and CockroachDB Self-Hosted v21.2. Topics specific to Serverless and Dedicated clusters are within the new top-level user task categories. CockroachDB Cloud release notes are under Reference.
```